src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -801,7 +779,6 @@ def testFileDeletion(self):
                     # NB: the fooStream() methods return context managers
                     self.assertRaises(NoSuchFileException, self.jobstore_initialized.readFileStream(fileID).__enter__)
 
-        @slow
```

Comment:
Is this no longer slow?

Reply:
It's still slow.  Just cruft since I didn't expect a review yet.  I'll add the WIP.